tests/draft-next/refRemote.json

```diff
@@ -99,7 +99,40 @@
         ]
     },
     {
-        "description": "base URI change - change folder in subschema",
+        "description": "base URI change - change folder in subschema with root ref",
+        "schema": {
+            "$id": "http://localhost:1234/scope_change_defs2.json",
+            "type" : "object",
+            "properties": {
+                "list": {"$ref": "#/definitions/baz/definitions/bar"}
```

Comment:
This should be changed to `$defs`, and given its own file in remotes. Because the identifier is hidden in `definitions`, my implementation doesn't see it and it can't resolve this reference.